parsing/pcoq.mli

```diff
@@ -123,24 +123,29 @@ val parse_string : 'a Entry.t -> ?loc:Loc.t -> string -> 'a
 val eoi_entry : 'a Entry.t -> 'a Entry.t
 val map_entry : ('a -> 'b) -> 'a Entry.t -> 'b Entry.t
 
-type gram_universe
+type gram_universe [@@deprecated "Deprecated in 8.13"]
+[@@@ocaml.warning "-3"]
+val get_univ : string -> gram_universe [@@deprecated "Deprecated in 8.13"]
+val create_universe : string -> gram_universe [@@deprecated "Deprecated in 8.13"]
 
-val get_univ : string -> gram_universe
-val create_universe : string -> gram_universe
+val new_entry : gram_universe -> string -> 'a Entry.t [@@deprecated "Deprecated in 8.13"]
 
-val new_entry : gram_universe -> string -> 'a Entry.t
+val uprim : gram_universe [@@deprecated "Deprecated in 8.13"]
+val uconstr : gram_universe [@@deprecated "Deprecated in 8.13"]
+val utactic : gram_universe [@@deprecated "Deprecated in 8.13"]
 
-val uprim : gram_universe
-val uconstr : gram_universe
-val utactic : gram_universe
+val create_generic_entry : gram_universe -> string ->
+  ('a, rlevel) abstract_argument_type -> 'a Entry.t
+  [@@deprecated "Deprecated in 8.13.  Use create_generic_entry2 instead."]
+[@@@ocaml.warning "+3"]
+
+val create_generic_entry2 : string ->
+  ('a, rlevel) abstract_argument_type -> 'a Entry.t
```

Comment:
I'm not a big fan of the `foo2` naming scheme. Plugin compatibility is anyway rarely preserved from a Coq version to the other, so, I would just reuse the same name `create_generic_entry`.

Reply:
If I can easily prevent breaking a plugin, I would always choose to do so.  Adding a `2` on the end seems a small price to do so.  Many IDEs show possible completions when you type in a prefix.
Changing the signature breaks a couple of the plugins in CI, so IIUC it would require creating overlays.  Though a little inelegant, can we live with an extra `2` here?  WDYT?

Reply:
Alternatively, I could keep the current routine name and just ignore the universe argument.